great_expectations/data_context/checkpoint_template.yml

```diff
@@ -0,0 +1,21 @@
+# This checkpoint was created by the command `great_expectations checkpoint new`.
+#
+# A checkpoint is a list of one or more batches paired with one or more
+# Expectation Suites and a configurable Validation Operator.
+#
+# It can be run with the `great_expectations checkpoint run` command.
+# You can edit this file to add batches of data and expectation suites.
+#
+# For more details please see
+# https://docs.greatexpectations.io/en/latest/command_line.html#great-expectations-checkpoint-new-checkpoint-suite
+validation_operator_name: action_list_operator
```

Comment:
would be good if any of those parameters could be overwritten through the CLI

for example it'll be common to have one validation operator for local development and one for production (publishing to S3)

Also we have 2 environments where tables have slightly different naming, so we would need to be able to overwrite the batch-kwargs through the CLI as well

Reply:
The yml could be templated to allow injecting some variables into it as well ?

`action_list_operator_${env}`

table: `${env}_schema.table`

Reply:
I love both of these ideas. I think I can fit in the CLI validation operator override before I ship this. If I don't get that in today both of these ideas would be great issues to track as well!